securedrop/source_templates/offline.html

```diff
@@ -0,0 +1,3 @@
+{% extends "base.html" %}
+
+{% block tab_title %}{{ gettext('Server Error') }}{% endblock %}
```

Comment:
Apparently this should be "Server error" (note capitalization) to avoid awaking the translation gods.

Reply:
Done.